README.md

```diff
@@ -1,179 +1,25 @@
-# Experiments for Faabric Paper
+# Faabric Experiments
 
-This repository is the common entrypoint for all experiments in [faabric](
-    https://github.com/faasm/faabric).
+This repository is a skeleton for the experiment repos for
+[Faasm](http://github.com/faasm/faasm) and 
+[Faabric](https://github.com/faasm/faabric).
 
-## Table of Contents
+For information on accessing and provisioning Azure resources see [the
+docs](docs/azure.md).
 
-1. [Experiment Index](#experiment-index)
-2. [Quick Start](#quick-start)
-3. [First Time Users](#first-time-users)
-4. [Creating a New Experiment](#new-experiment) 
-5. [Cluster Provisioning](#cluster-provisioning)
-6. [Troubleshooting](#troubleshooting)
+If creating a new experiment, see [the docs](docs/new_experiments.md).
 
-## Experiment Index <a name="experiment-index">
+## Experiments
 
 | Name | Status | Results | Run Experiment |
 | --- | --- | --- | --- |
 | [LAMMPS](https://github.com/faasm/experiment-lammps) | :x: MPI Error | [Plots](https://github.com/faasm/experiment-base/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-lammps/blob/master/README.md) |
 | [ParRes Kernels](https://github.com/faasm/experiment-kernels) | :heavy_check_mark: Waiting Plot Spec | [Plots](https://github.com/faasm/experiment-base/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-kernels/blob/master/README.md) |
 | [PyWren](https://github.com/faasm/experiment-pywren) | :clock9: WIP | [Plots](https://github.com/faasm/experiment-master/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-pywren/blob/master/README.md) |
 
-## Quick Start <a name="quick-start">
+## Quick Start
 
-If its the first time you are running these experiments, **you must first follow
-the [instructions for first time users](#first-time-users)**.
-
-Then, you may navigate to any of the experiments repository (see the [Experiment
-Index](#experiment-index)) and run a specific experiment.
-
-Alternatively, you may run all experiments in the paper by doing:
 ```
 TODO: have a one-liner to run all the scripts prior to the artifact evaluation
 ```
```

Comment:
A one-liner may be a bit adventurous, but it will probably look something like:

- Pull a Docker image
- Run a script to create a new container
- Enter the credentials you've been given
- Run some other script to execute all the experiments
- Inspect the graphs